lm75bd/lm75bd.c

```diff
@@ -1,6 +1,6 @@
 #include "lm75bd.h"
 #include "i2c_io.h"
-#include "errors.h"
+#include "../sys/errors.h"
```

Comment:
you shouldn't need to do a relative include here since the cmake is already set up with the proper include directories